core/api_files.go

```diff
@@ -190,3 +190,33 @@ func (a *api) lsThreadFileTargetKeys(g *gin.Context) {
 
 	pbJSON(g, http.StatusOK, keys)
 }
+
+// lsThreadFileTargetKeys godoc
```

Comment:
these generate the swagger docs, so would need to be updated and then run `make docs`